### PR TITLE
Assign PostgreSQL userName to correct attribute

### PR DESCRIPTION
**Pull request by tsndqst**
Signed-off-by: Tim Sandquist <tim.sandquist@gmail.com>

Fix problem with PostgreSQL scaler introduced by https://github.com/kedacore/keda/pull/2114.  The userName overwrites the port value when creating the connection string from individual values.

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work))
- [x] Changelog has been updated

Fixes https://github.com/kedacore/keda/issues/2430

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -30,6 +30,7 @@
 - Graphite Scaler: use the latest datapoint returned, not the earliest ([#2365](https://github.com/kedacore/keda/pull/2365))
 - Kubernetes Workload Scaler: ignore terminated pods ([#2384](https://github.com/kedacore/keda/pull/2384))
 - `keda-operator` Cluster Role: add `list` and `watch` access to service accounts ([#2406](https://github.com/kedacore/keda/pull/2406))|([#2410](https://github.com/kedacore/keda/pull/2410))
+- PostgreSQL Scaler: Assign PostgreSQL userName to correct attribute ([#2432](https://github.com/kedacore/keda/pull/2432))
```
Comment (zroubalik) [inline CHANGELOG.md:33]:
```suggestion
- PostgreSQL Scaler: Assign PostgreSQL `userName` to correct attribute ([#2432](https://github.com/kedacore/keda/pull/2432))
```

**Review by zroubalik — COMMENTED** [review]
Looking good, could you please tweak the Changelog and also fix the merge conflict in Changelog?

**Review by JorTurFer — APPROVED** [review]

**Comment by JorTurFer** [comment]
Nice catch!
Could you add a test for it?

**Pull request merged**